README.md

```diff
@@ -179,7 +179,7 @@ Below is a list of sites using Lanyard right now, check them out! A lot of them
 - [crugg.de](https://crugg.de)
 - [igalaxy.dev](https://igalaxy.dev)
 - [itspolar.dev](https://itspolar.dev)
-- [v30xy.github.io](https://v30xy.github.io)
+- [v30xy.github.io](https://v30xy.gitlab.io)
```

Comment:
Shouldn't the link text say `v30xy.gitlab.io` instead of `v30xy.github.io` ?

Reply:
from GitHub to GitLab